<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,6,0,1">
        <item>
         <widget class="QDateEdit" name="dateEditActual">
          <property name="font">
           <font>
            <family>Segoe UI Semibold</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::Section::MonthSection</enum>
          </property>
          <property name="displayFormat">
           <string>MMMM dd, yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDebug">
          <property name="text">
           <string>Debug</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonNuevaReserva">
          <property name="text">
           <string>Nueva Reserva</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Información:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Reservas</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="tabWidgetReservas">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::TabFocus</enum>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::TabPosition::North</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::TabShape::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <property name="elideMode">
           <enum>Qt::TextElideMode::ElideNone</enum>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <property name="movable">
           <bool>false</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tabLlegadas">
           <attribute name="title">
            <string>Llegadas</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTabWidget" name="tabWidgetLlegadas">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tabLlegadasHoy">
               <attribute name="title">
                <string>Hoy</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QTableWidget" name="tableWidgetLlegadasHoy">
                  <property name="rowCount">
                   <number>0</number>
                  </property>
                  <property name="columnCount">
                   <number>3</number>
                  </property>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>200</number>
                  </attribute>
                  <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderStretchLastSection">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Cliente</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string># Confirmación</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Habitación</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tabLlegadasManiana">
               <attribute name="title">
                <string>Mañana</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QTableWidget" name="tableWidgetLlegadasManiana">
                  <property name="rowCount">
                   <number>0</number>
                  </property>
                  <property name="columnCount">
                   <number>3</number>
                  </property>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>200</number>
                  </attribute>
                  <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderStretchLastSection">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Cliente</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string># Confirmación</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Habitación</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabSalidas">
           <attribute name="title">
            <string>Salidas</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTabWidget" name="tabWidgetSalidas">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tabSalidasHoy">
               <attribute name="title">
                <string>Hoy</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QTableWidget" name="tableWidgetSalidasHoy">
                  <property name="rowCount">
                   <number>0</number>
                  </property>
                  <property name="columnCount">
                   <number>3</number>
                  </property>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>200</number>
                  </attribute>
                  <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderStretchLastSection">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Cliente</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string># Confirmación</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Habitación</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tabSalidasManiana">
               <attribute name="title">
                <string>Mañana</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QTableWidget" name="tableWidgetSalidasManiana">
                  <property name="rowCount">
                   <number>0</number>
                  </property>
                  <property name="columnCount">
                   <number>3</number>
                  </property>
                  <attribute name="horizontalHeaderCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>200</number>
                  </attribute>
                  <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderStretchLastSection">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Cliente</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string># Confirmación</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Habitación</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabOcupacion">
           <attribute name="title">
            <string>Ocupación Actual</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QTableWidget" name="tableWidgetOcupacion">
              <property name="rowCount">
               <number>0</number>
              </property>
              <property name="columnCount">
               <number>5</number>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>100</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Habitación</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Cliente</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fecha Salida</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Importe</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTabWidget" name="tabWidgetInformacion">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabInformacionClientes">
           <attribute name="title">
            <string>Clientes</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QTableWidget" name="tableWidgetClientes">
              <property name="columnCount">
               <number>7</number>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>82</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>ID:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Nombre:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Email:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Teléfono:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Nacionalidad:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string># Estancias:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonNuevoCliente">
                <property name="text">
                 <string>Nuevo Cliente</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabInformacionHabitaciones">
           <attribute name="title">
            <string>Habitaciones</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QTableWidget" name="tableWidgetHabitaciones">
              <property name="columnCount">
               <number>8</number>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>70</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Número:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Tipo:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Tamaño:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Camas:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Coste:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Disponible:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>En Arreglos:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonNuevaHabitacion">
                <property name="text">
                 <string>Nueva Habitación</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabInformacionReservas">
           <attribute name="title">
            <string>Reservas</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QTableWidget" name="tableWidgetReservas">
              <property name="columnCount">
               <number>7</number>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>90</number>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string># Reserva:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Cliente:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Habitación:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Estado:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fecha Inicio:</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_adir">
    <property name="title">
     <string>Añadir</string>
    </property>
    <addaction name="actionCliente"/>
    <addaction name="actionHabitacion"/>
    <addaction name="actionReserva"/>
   </widget>
   <widget class="QMenu" name="menuRegistrar">
    <property name="title">
     <string>Registrar</string>
    </property>
    <addaction name="actionEntrada"/>
    <addaction name="actionSalida"/>
   </widget>
   <addaction name="menuA_adir"/>
   <addaction name="menuRegistrar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCliente">
   <property name="text">
    <string>Cliente</string>
   </property>
  </action>
  <action name="actionHabitacion">
   <property name="text">
    <string>Habitación</string>
   </property>
  </action>
  <action name="actionReserva">
   <property name="text">
    <string>Reserva</string>
   </property>
  </action>
  <action name="actionEntrada">
   <property name="text">
    <string>Entrada</string>
   </property>
  </action>
  <action name="actionSalida">
   <property name="text">
    <string>Salida</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
